.eslintrc.js

```diff
@@ -25,5 +25,7 @@ module.exports = {
     "no-param-reassign": 1,
     "arrow-parens": [2, "as-needed"],
     "prefer-object-spread": 0,
+    "no-param-reassign": 0,
+    "no-unused-vars": 0,
```

Comment:
Don't put this rule here (for all files). Just disable there `mockingcase` is declared

Reply:
All good, but I've changed `no-param-reassign` to be 0. See above comment.